src/redux/VehicleSlice/index.tsx

```diff
@@ -1,15 +1,25 @@
 import { createSlice } from "@reduxjs/toolkit";
 import { VehicleList } from "./Vehicletypes";
 import { getVehicleActions } from "./VehicleAsyncThunk";
+import constant from "../../config/constant";
 
 const initialState: VehicleList = {
   list: [],
   isLoading: false,
+  page: constant.page.defaultNumber,
+  total: constant.page.defaultTotal,
+  nextPageUrl: null,
+  prevPageUrl: null,
```

Comment:
Please explain me the logic behind this ?

Reply:
I removed All the issue